www/IndoorAtlas.js

```diff
@@ -1,470 +1,521 @@
+    var win = function(p) {
+      var floorplan = new FloorPlan(
+        p.id,
+        p.name,
+        p.url,
+        p.floorLevel,
+        p.bearing,
+        p.bitmapHeight,
+        p.bitmapWidth,
+        p.heightMeters,
+        p.widthMeters,
+        p.metersToPixels,
+        p.pixelsToMeters,
+        p.bottomLeft,
+        p.center,
+        p.topLeft,
+        p.topRight
+      );
+      successCallback(floorplan);
+    };
+    var fail = function(e) {
+      var err = new PositionError(e.code, e.message);
+      if (errorCallback) {
+        errorCallback(err);
+      }
+    };
+    exec(win, fail, "IndoorAtlas", "fetchFloorplan", [floorplanId]);
+  },
+
+  coordinateToPoint: function(coords, floorplanId, successCallback, errorCallback){
+    var win = function(p) {
+      successCallback(p);
+    };
+    var fail = function(e) {
+      var err = new PositionError(e.code, e.message);
+      if (errorCallback) {
+        errorCallback(err);
+      }
+    };
+    exec(win, fail, "IndoorAtlas", "coordinateToPoint",
+    [coords.latitude, coords.longitude, floorplanId]);
+  },
+
+  pointToCoordinate: function(point, floorplanId, successCallback, errorCallback) {
+    var win = function(p) {
+      successCallback(p);
+    };
+    var fail = function(e) {
+      var err = new PositionError(e.code, e.message);
+      if (errorCallback) {
+        errorCallback(err);
+      }
+    };
+    exec(win, fail, "IndoorAtlas", "pointToCoordinate",
+    [point.x, point.y, floorplanId]);
+  },
+
+  setDistanceFilter: function(successCallback, errorCallback, distance) {
+    var win = function(p) {
+      successCallback(p);
+    };
+    var fail = function(e) {
+      if (errorCallback) {
+        errorCallback(e);
+      }
+    };
+    exec(win, fail, "IndoorAtlas", "setDistanceFilter", [distance.distance]);
+  },
+
+  setSensitivities: function(successCallback, errorCallback, sensitivity) {
+    var win = function(success) {
+      successCallback(success)
+    };
+    var fail = function(e) {
+      if (errorCallback) {
+        errorCallback(e);
+      }
+    };
+    exec(win, fail, "IndoorAtlas", "setSensitivities", [sensitivity.orientationSensitivity, sensitivity.headingSensitivity]);
+  },
+
+  getFloorCertainty: function(successCallback, errorCallback) {
+    var win = function(p) {
+      successCallback(p);
+    };
+    var fail = function(e) {
+      if (errorCallback) {
+        errorCallback(e);
+      }
+    };
+    exec(win, fail, "IndoorAtlas", "getFloorCertainty");
+  },
+
+  getTraceId: function(successCallback, errorCallback) {
+    var win = function(p) {
+      successCallback(p);
+    };
+    var fail = function(e) {
+      if (errorCallback) {
+        errorCallback(e);
+      }
+    };
+    exec(win, fail, "IndoorAtlas", "getTraceId");
+  },
+
+  /**
+   * Initialize graph with the given graph JSON
+   */
+  initWithGraph: function(graphJson) {
```

Comment:
This should be renamed: Now it looks like `IndoorAtlas.initWithGraph` as if you would initialize IndoorAtlas with a graph, whereas it should clearly indicate that you are initializing _wayfinding_ with a graph.

A better name would be, for example, `IndoorAtlas.buildWayfinder`